package/yast2-network.changes

```diff
@@ -1,3 +1,12 @@
+-------------------------------------------------------------------
+Tue Oct 13 12:57:36 UTC 2020 - Michal Filka <mfilka@suse.com>
+
+- bnc#1175360
```

Comment:
what about also mention https://bugzilla.suse.com/show_bug.cgi?id=1175206 ? at least it is also mentioned already in the PR description

Reply:
will add ...